markdown/chapters/univariate-analysis/main.rmd

```diff
@@ -0,0 +1,1672 @@
+
+Finally consider the three distributions together.
+
+```{r, fig.cap = "Density plot showing distributions of responses regarding all three plots, after removing values greater or equal to 1000 from the responses fore the logarithmic plot"}
+
+brks <- c("Control", "Truncated", "Logarithmic")
+vals <- c("#1c9e77", "#d95f02", "#7570b3")
+
+ggplot() +
+  geom_density(data = as.data.frame(control), aes(x=control, col = "Control"))+
+  geom_density(data = as.data.frame(truncated), aes(x=truncated, col = "Truncated"))+
+  geom_density(data = as.data.frame(logarithmic), aes(x=logarithmic, col = "Logarithmic"))+
+  labs(x="Response", y="Density")+
+  scale_colour_manual(name = " ", breaks = brks, values = vals)+
+  theme_classic()
+
+```
+
+The distribution looks remarkably different to the other two, with much less height and a wider spread. 
+
+Overall, it appears that the use of the truncated scale had little impact on judging the height of an individual bar as compared to a control with no scale alterations, and if anything marginally improved the respondents' ability to judge the height. It has also been observed that the scale by default ending at 40 while the bar height is at 42 leads to some underestimation in the height for both the control and truncated bar plots.
+
+Now considering the logarithmic-scale plot, The Python default of standard form notation appears to have confused certain respondents, who are perhaps not as used to seeing this notation, and there was a very large range in the responses along with one person not even entering a number, but rather stating that they "Don't know", and another stating they believed the value was "Next to none". The "Next to none" entry is very subjective, but could potentially be be assumed as a value close to 0, once again maybe as a result of standard form being less well known to this respondent.
+
+This should be considered when designing visualisations; the creator of the visualisations may find the logarithmic scale or standard form more effective in showing the data, but they should consider the target audience. Are the audience going to be familiar with this? If, for example, visualisations are being published in a paper targeted at academics in a subject likely to use such scalings often and understand them, this may be a good way to depict the data. However, using this in something such as an advertising campaign could mislead the public, causing them to either over or under estimate values. As previously discussed, however, this is often done deliberately in order to push the message the creator wishes to sell. 
```

Comment:
If/when you do the focus group this is a particular point to discuss, are you conclusions as to the causation behind correct here? (Not saying they're not but it's something to potentially ask respondents). If you do then you should point at that here.